pcsx2/IopBios.cpp

```diff
@@ -100,632 +79,906 @@ namespace R3000A {
+			"INT_34", "INT_35", "INT_36", "INT_37",                    //34
+			"INT_38", "INT_39", "INT_3A", "INT_3B",                    //38
+			"INT_3C", "INT_3D", "INT_3E", "INT_3F",                    //3C
+			"INT_MAX"                                                  //40
+		};
+		// clang-format on
 
-namespace intrman {
-	static const char* intrname[] = {
-		"INT_VBLANK",   "INT_GM",       "INT_CDROM",   "INT_DMA",		//00
-		"INT_RTC0",     "INT_RTC1",     "INT_RTC2",    "INT_SIO0",		//04
-		"INT_SIO1",     "INT_SPU",      "INT_PIO",     "INT_EVBLANK",	//08
-		"INT_DVD",      "INT_PCMCIA",   "INT_RTC3",    "INT_RTC4",		//0C
-		"INT_RTC5",     "INT_SIO2",     "INT_HTR0",    "INT_HTR1",		//10
-		"INT_HTR2",     "INT_HTR3",     "INT_USB",     "INT_EXTR",		//14
-		"INT_FWRE",     "INT_FDMA",     "INT_1A",      "INT_1B",		//18
-		"INT_1C",       "INT_1D",       "INT_1E",      "INT_1F",		//1C
-		"INT_dmaMDECi", "INT_dmaMDECo", "INT_dmaGPU",  "INT_dmaCD",		//20
-		"INT_dmaSPU",   "INT_dmaPIO",   "INT_dmaOTC",  "INT_dmaBERR",	//24
-		"INT_dmaSPU2",  "INT_dma8",     "INT_dmaSIF0", "INT_dmaSIF1",	//28
-		"INT_dmaSIO2i", "INT_dmaSIO2o", "INT_2E",      "INT_2F",		//2C
-		"INT_30",       "INT_31",       "INT_32",      "INT_33",		//30
-		"INT_34",       "INT_35",       "INT_36",      "INT_37",		//34
-		"INT_38",       "INT_39",       "INT_3A",      "INT_3B",		//38
-		"INT_3C",       "INT_3D",       "INT_3E",      "INT_3F",		//3C
-		"INT_MAX"														//40
-	};
+		void RegisterIntrHandler_DEBUG()
+		{
+			DevCon.WriteLn(Color_Gray, "RegisterIntrHandler: intr %s, handler %x", intrname[a0], a2);
+		}
+	} // namespace intrman
 
-	void RegisterIntrHandler_DEBUG()
+	namespace sifcmd
 	{
-		DevCon.WriteLn(Color_Gray, "RegisterIntrHandler: intr %s, handler %x", intrname[a0], a2);
-	}
-}
+		void sceSifRegisterRpc_DEBUG()
+		{
+			DevCon.WriteLn(Color_Gray, "sifcmd sceSifRegisterRpc: rpc_id %x", a1);
+		}
+	} // namespace sifcmd
 
-namespace sifcmd {
-	void sceSifRegisterRpc_DEBUG()
+	u32 irxImportTableAddr(u32 entrypc)
 	{
-		DevCon.WriteLn( Color_Gray, "sifcmd sceSifRegisterRpc: rpc_id %x", a1);
-	}
-}
+		u32 i;
 
-u32 irxImportTableAddr(u32 entrypc)
-{
-	u32 i;
+		i = entrypc - 0x18;
+		while (entrypc - i < 0x2000)
+		{
+			if (iopMemRead32(i) == 0x41e00000)
+				return i;
+			i -= 4;
+		}
 
-	i = entrypc - 0x18;
-	while (entrypc - i < 0x2000) {
-		if (iopMemRead32(i) == 0x41e00000)
-			return i;
-		i -= 4;
+		return 0;
 	}
 
-	return 0;
-}
+	const char* irxImportFuncname(const std::string& libname, u16 index)
+	{
+#include "IopModuleNames.cpp"
 
-const char* irxImportFuncname(const std::string &libname, u16 index)
-{
-	#include "IopModuleNames.cpp"
+		switch (index)
+		{
+			case 0:
+				return "start";
+			// case 1: reinit?
+			case 2:
+				return "shutdown";
+				// case 3: ???
+		}
 
-	switch (index) {
-		case 0: return "start";
-		// case 1: reinit?
-		case 2: return "shutdown";
-		// case 3: ???
+		return 0;
 	}
 
-	return 0;
-}
+// clang-format off
```

Comment:
the formatting is off there

Reply:
Hello @GovanifY ,
It was done in purpose as was recommended by @tellowkrinkle , because otherwise incoming macro `MODULE` wasn't properly aligned.

Thanks

Reply:
I get that, but it still isn't aligned here!
![screenshot](https://user-images.githubusercontent.com/6375438/117680950-27c4ec00-b1a1-11eb-972d-ac1e6885a6a7.png)

Is this intended? Doesn't seem that readable to me...

Reply:
You were totally right!
I have fixed it and put it how it was before, much cleaner.

```cpp
// clang-format off
#define MODULE(n) if (#n == libname) { using namespace n; switch (index) {
#define END_MODULE }}
#define EXPORT_D(i, n) case (i): return n ## _DEBUG;
#define EXPORT_H(i, n) case (i): return n ## _HLE;
// clang-format on
```

Thanks @GovanifY